docs/releases/2.2.txt

```diff
@@ -33,6 +33,9 @@ What's new in Django 2.2
 Minor features
 --------------
 
+* Added support for mangled methods and made ``name`` argument optional for the
```

Comment:
I'm not sure what Adam was thinking of with regards to the release note, but we don't usually document changes in `django.utils` to be "minor features".

Reply:
Do you know where it should go?

Reply:
Most likely it doesn't need a release unless there's some backwards incompatible change.

Reply:
I've seen a lot of use of `@cached_property` so I thought it would count, but yeah let's not worry about it.

Reply:
removed